### DISCUSSION TITLE
How to import to a subcollection

### DISCUSSION THREAD

**Discussion by gtlsgamr** [benyap/firestore-backfire#435] [Q&A] [answered]
I am trying to export a top-level collection `foo` which exports just fine. 
Next, I want to import this exported json into a second level collection, which is `foo/baz`.

I am getting the below error
` Error: Value for argument "documentPath" must point to a document, but was "foo/baz/bar/bat/23/taz/1wda". Your path does not contain an even number of components.
`

Below is the command I am using to export
`backfire export ./testdata  --config config --paths foo --debug  `

And to import

`backfire import ./testdata.ndjson --config config --gcpProject project  --debug `

FYI, in the json I am editing all the paths that begin with `foo` with `foo/baz`

**Comment by benyap** [comment, marked as answer]
Hi @gtlsgamr, the error should be fairly self explanatory, and it comes from the Firebase SDK. Please take a look at the [Cloud Firestore Data model](https://firebase.google.com/docs/firestore/data-model) to understand how documents and collections work.

By changing `foo` to `foo/baz`, you've turned your paths, which would have been originally pointing to documents, to be pointing to a collection.

For example, `users/john` would have pointed to a document with the ID `john` in the collection `users`. You've now changed the path to `users/baz/john`, which is now pointing to a _subcollection_ called `john` under the document `baz`. You cannot import a document to to a collection path.

If you want to change the nesting level of your documents, you'll need to add an **even** number of segments.

Closing this as it's not an issue with the package, but feel free to reply if you need further assistance!
